frontend/src/mocks-server/__mockdata__/alerts.ts

```diff
@@ -0,0 +1,49 @@
+export const allAlertChannels = [
+	{
```

Comment:
In below responses, just make sure once that you are not exposing any companies info, and if in doubt replace it with random and test fillers

Reply:
Done :+1: . makes sense :slightly_smiling_face: